### ISSUE TITLE
[Website] (a11y) missing lang attribute on routes

### ISSUE THREAD

**Issue by noelmace** [fullwebdev/fullwebdev#25]
# Website - 🐞 bug report

When a route content language is different from the app language, the lang attribute isn't set.

## Description

When a page can't be found for the selected language, the router falls back to the content associated with the default language. As the lang attribute is set on the documentElement according to the selected language, we need to also set it on "router-outlet" when its content language is different from the selected one, for accessibility reasons.

## Expected Behavior

Set the lang attribute on "router-outlet" when the "page" language is different from the selected language.
